CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- fixed: (Fantom) Updated down ftmscan explorer to https://explorer.fantom.network/
```

Comment:
The explorer was already changed from ftmscan to ftmscout recently. I'll leave it up to you to either update the changelog and merge or delete this PR.

Reply:
Deleting PR. Ftmscout looks more etherscan-like